employee-benefits/pto-canada.md

```diff
@@ -2,7 +2,7 @@
 title: Benefits policy (Canadian employees)
```

Comment:
@ContentBrewmaster can you update this title to match the US version as well?

Reply:
Done!